parm/nems.configure.medcmeps_atm_ocn_ice.IN

```diff
@@ -42,7 +42,7 @@ ICE_attributes::
   ProfileMemory = false
   OverwriteSlice = true
   mesh_ice = @[MESHICE]
-  stop_n = @[FHMAX]
+  stop_n = @[FV3_RESTART_H]
```

Comment:
I am curious what is the stop_n in ice, why is it set to fv3 restart time? In general model will continue integration while writing restart

Reply:
For the ice,ocean and mediator, if you want to integrate 12 hours after a restart, it is a simple 12h from the restart time.  But in the restart test for FV3, the FHMAX=24, FHROT=12 for example. But we only want the ocean and ice to integrate 12h, not 24. If we're not restarting, the other components can use FHMAX to determine their stop time and that is how FV3_RESTART_H is set (=FHMAX).

Reply:
Does the final integration time get overwritten by CICE by the driver in the end anyways?

Reply:
I don't think so. I think it is a hard stop. CICE uses the value to set a stop alarm.

Reply:
I thought start and stop times were overwritten by values from the driver? That that was part of NUOPC... but ? I know that's true for WW3 and I thought for MOM6 as well.

Reply:
Actually I was wrong about what stop_n does. Currently it is not an optional config setting but I hadn't looked carefully at what it actual controls. The only function is to have the timing code inside of cice turn on and off. We should actually make it optionally configurable so that we don't need to have it present in nems.configure.

Reply:
I agree. If we do need to set ICE integration length, it would be "FHMAX-FHROT". In general, each subcomponent inherits the clock from top level component to control integration time. So I think the  "The only function is to have the timing code inside of cice turn on and off" makes sense to me. But we can make changes later.